projectHelpers.js

```diff
@@ -44,6 +44,28 @@ const getAndroidRuntimeVersion = (projectDir) => {
     }
 }
 
+const getWebpackConfig = (projectDir, env, configPath = "webpack.config.js") => {
+    const configAbsolutePath = path.resolve(projectDir, configPath);
+    let config;
+    try {
+        config = require(configAbsolutePath);
+    } catch (e) {
+        throw new Error(
+            `Couldn't load webpack config from ${configAbsolutePath}. ` +
+            `Original error:\n${e}`
```

Comment:
Not a merge-stopper but I'd rather we use `require('os').EOL` instead of a hardcoded string here